hips/draw/simple.py

```diff
@@ -1,5 +1,156 @@
+        self.wcs = WCS(tile.header)
+        self.corners = None
+        self.pt = None
+
+    def compute_corners(self) -> None:
+        nside = hp.order2nside(self.tile.meta.order)
+        theta, phi = boundaries(nside, self.tile.meta.ipix)
+        radec = SkyCoord(ra=phi, dec=np.pi / 2 - theta, unit='radian', frame='icrs')
+        self.corners = []
+        for i in range(len(radec.ra.deg)):
+            self.corners.append([radec.ra.deg[i], radec.dec.deg[i]])
+
+    def apply_projection(self) -> None:
+        src = self.geometry.wcs.wcs_world2pix(self.corners, 0)
+        dst = np.array([[511, 0], [511, 511], [0, 511], [0, 0]])
+        self.pt = tf.ProjectiveTransform()
+        self.pt.estimate(src, dst)
+
+    def warp_image(self) -> np.ndarray:
+        return tf.warp(self.tile.data, self.pt, output_shape=self.shape)
+
+
+def _fetch_tiles(healpix_pixel_indices: np.ndarray, order: int, hips_survey: HipsSurveyProperties) -> 'HipsTile':
+    """Fetch HiPS tiles from a remote URL.
+
+    Parameters
+    ----------
+    healpix_pixel_indices : np.ndarray
+        A list of HEALPix pixel indices
+    order : int
+        Order of the HEALPix map
+    hips_survey : HipsSurveyProperties
+        An object of HipsSurveyProperties
+
+    Returns
+    -------
+    'HipsTile'
+        Returns an object of  HipsTile
+    """
+    for healpix_pixel_index in healpix_pixel_indices:
+        tile_meta = HipsTileMeta(order=order, ipix=healpix_pixel_index, file_format='fits')
+        tile = HipsTile.fetch(tile_meta, hips_survey.base_url)
+        yield tile
+
+
+def make_sky_image(geometry: WCSGeometry, hips_survey: HipsSurveyProperties) -> np.ndarray:
+    """Make sky image: fetch tiles and draw.
+
+    Parameters
+    ----------
+    geometry : `~hips.utils.WCSGeometry`
+        Geometry of the output image
+    hips_survey : `~hips.HipsSurveyProperties`
+        HiPS survey properties
+
+    Returns
+    -------
+    data : `~numpy.ndarray`
+        Output image pixels
+
+    Examples
+    --------
+    >>> from astropy.io import fits
+    >>> from hips.utils import WCSGeometry
+    >>> from hips.draw import make_sky_image
+    >>> from hips.tiles import HipsSurveyProperties
+    >>> geometry = WCSGeometry.create(
+    ...     skydir=SkyCoord(0, 0, unit='deg', frame='galactic'),
+    ...     shape=(1000, 2000), coordsys='GAL',
+    ...     projection='AIT', cdelt=0.01, crpix=(1000, 500),
+    ... )
+    >>> url = 'https://raw.githubusercontent.com/hipspy/hips-extra/master/datasets/samples/DSS2Red/properties'
+    >>> hips_survey = HipsSurveyProperties.fetch(url)
+    >>> data = make_sky_image(geometry, hips_survey)
+    >>> hdu = fits.PrimaryHDU(data=data, header=geometry.header)
+    >>> hdu.writeto('my_image.fits')
+    """
+    healpix_pixel_indices = compute_healpix_pixel_indices(geometry, hips_survey.hips_order)
+    path = Path(os.environ['HIPS_EXTRA'])
+    tiles_path = path / 'datasets' / 'samples' / 'DSS2Red' / 'Norder3' / 'Dir0'
```

Comment:
Clearly hard-coding "DSS2Red" isn't a good solution.
Can't you change the API of this function so that the user can select what they want?
Alternatively, if you want to do that in a follow-up PR, at least leave a "TODO" comment here.